src/routes/api-documentation.ts

```diff
@@ -0,0 +1,44 @@
+import Router from '@koa/router';
+import { Context, Middleware, Next } from 'koa';
+import { hydra, owl, schema } from 'rdf-namespaces';
+import Routes from './index';
+
+export default (router: Router): Middleware => (
+  async ({ request, response }: Context, next: Next): Promise<void> => {
+    response.body = {
+      '@context': {
+        '@base': request.origin,
+      },
+      '@id': router.url(Routes.ApiDocumentation),
+      '@type': hydra.ApiDocumentation,
+      [hydra.entrypoint]: { '@id': router.url(Routes.EntryPoint) },
+      [hydra.supportedClass]: [
+        {
+          '@id': schema.EntryPoint,
+          '@type': hydra.Class,
+          [hydra.title]: { '@value': 'API entry point', '@language': 'en' },
+          [hydra.supportedOperation]: {
+            '@type': hydra.Operation,
+            [hydra.title]: { '@value': 'Get the entry point', '@language': 'en' },
+            [hydra.method]: { '@value': 'GET' },
+            [hydra.expects]: { '@id': owl.Nothing },
```

Comment:
I know your OWL allergy but this may be the best explicit way to state that an operation really does not want a payload. This is also how the original issue tracker example was implemented I think

Reply:
I'm still not convinced - I feel i.e. empty RDF list still would be better, but I won't die for it. Just saying that OWL is not widely understood on client side

Reply:
What is not to understand. I literally says "expects nothing"? 🤣

Reply:
Indeed - client that does not know what _owl_ is it will try to create instances of class `owl:Nothing`!